servo/types.py

```diff
@@ -529,10 +529,10 @@ class Unit(str, enum.Enum):
     count = ""
     rate = ""
     percentage = "%"
-    milliseconds = "ms"
+    milliseconds = "milliseconds"
     bytes = "bytes"
     requests_per_minute = "rpm"
-    requests_per_second = "rps"
+    requests_per_second = "req / sec"
```

Comment:
I feel like we kinda have to define a standard here -- having `rpm` and `req / sec` feels dissonant.

I'm also a little iffy on if we want to introduce unit abbreviations that are this long. We're gonna append it on to log messages and string representations of the metrics for output and it's a lot of horizontal real estate for something that is pretty well understood at a glance.